src/Pim/Component/Catalog/Model/ProductUniqueData.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Pim\Component\Catalog\Model;
+
+/**
+ * @author    Julien Janvier <julien.janvier@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class ProductUniqueData implements ProductUniqueDataInterface
+{
+    /** @var int */
+    protected $id;
+
+    /** @var ProductInterface */
+    protected $product;
+
+    /** @var ProductValueInterface */
+    protected $value;
```

Comment:
Ok so I think I didn't understand this part, as you store the `$rawData`, why do you need to have the product value here?